watertap/tools/oli_api/credentials.py

```diff
@@ -102,6 +103,7 @@ def __init__(
         :param access_keys: list of access keys generated by user
         :param test: bool switch for automation during tests
         :param interactive_mode: bool to switch level of logging display from info to debug only
+        :param refresh: bool to use refresh token in login method
```

Comment:
What does this refresh actually do?

Reply:
From my recollection, it enables usage of a refresh token so that we don't log out prematurely.